modules/renter/hostdb/negotiate_test.go

```diff
@@ -259,3 +259,34 @@ func TestReviseContract(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+// TestUniqueHosts tests the UniqueHosts method of the pool type.
+// TODO: make pool safer to test, using real or mocked hostdb
```

Comment:
what do you mean by 'safer to test'? I am trying to stop merging TODOs into the master branch, either they should be done in the PR or it should be accepted that they don't really fit into the time/resource constraints of our development cycle.